pkg/orchestration/wiring/testdata/postgres.dependsons3.bundle.yaml

```diff
@@ -1,50 +0,0 @@
-apiVersion: smith.atlassian.com/v1
```

Comment:
After talking to @ash2k @nilebox and @marciogh we decided that the state should output and error, therefore I replaced this bundle with an error